File: backend/scratch_query.js
import './src/env.js';
import { supabaseAdmin } from './src/config/supabase.js';

async function queryReviews() {
  const { data, error } = await supabaseAdmin
    .from('reviews')
    .select('id, pr_number, status, error, updated_at, report_json')
    .eq('pr_number', 1)
    .order('updated_at', { ascending: false });
  console.log('Reviews for PR #1:');
  console.log(JSON.stringify(data, null, 2));
}

queryReviews();

File: backend/scratch_query.py
import os
import requests

supabase_url = "https://lwpcrygyvorggkijmlyq.supabase.co"
supabase_key = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6Imx3cGNyeWd5dm9yZ2draWptbHlxIiwicm9sZSI6InNlcnZpY2Vfcm9sZSIsImlhdCI6MTc4Mzc3OTMwMCwiZXhwIjoyMDk5MzU1MzAwfQ.ntzJzYGiv5ypdA0Hj1H_pjQ-2L1FxYmm2ilpuvL8v_c"

headers = {
    "apikey": supabase_key,
    "Authorization": f"Bearer {supabase_key}",
    "Content-Type": "application/json",
    "Prefer": "return=representation"
}

response = requests.get(
    f"{supabase_url}/rest/v1/reviews?pr_number=eq.4&select=id,pr_number,status,error,updated_at",
    headers=headers
)
print("Reviews for PR #4:")
print(response.json())
